assignments/arrays.js

```diff
@@ -75,30 +75,61 @@ let inventory = [
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
 
+ 
+  for(let i=0; i<inventory.length; i++){
+    if(inventory[i].id === 33){
+      console.log('Car 33 is a car ' + inventory[i].car_year  + ' ' + inventory[i].car_make + ' ' + inventory[i].car_model + ".")
+    }
+}
+  console.log();
```

Comment:
In this case, you don't need to go through every element of the array, you can just chose the one you want with `inventory[32]`